query-engine/schema-builder/Cargo.toml

```diff
@@ -9,10 +9,8 @@ schema = { path = "../schema" }
 prisma-models = { path = "../prisma-models" }
 once_cell = "1.3"
 
-[dev-dependencies.criterion]
-version = "0.4.0"
-default-features = false
-features = ["cargo_bench_support"]
+[dev-dependencies]
+codspeed-criterion-compat = "1.1.0"
```

Comment:
Does that change anything to how benchmarks run locally? I'm not too chuffed about replacing a known dependency with something less known that brings in its own transitive dependencies (see diff in Cargo.lock). But if it's the way forward, let's do it. I'll have a look at the source for codspeed-criterion-compat before approving.

Reply:
the codebase looks harmless enough, I'll try to run the benchmarks locally

Reply:
Ok I couldn't even notice a difference to vanilla criterion, let's do this :+1:

Reply:
Docs describe it as a harmless wrapper.

Reply:
I have trust issues with dependencies, I had to read the source :p